src/ci/scripts/free-disk-space.sh

```diff
@@ -14,11 +15,15 @@ printSeparationLine() {
 # compute available space
 # REF: https://unix.stackexchange.com/a/42049/60849
 # REF: https://stackoverflow.com/a/450821/408734
-getAvailableSpace() { echo $(df -a | awk 'NR > 1 {avail+=$4} END {print avail}'); }
+getAvailableSpace() {
+    df -a | awk 'NR > 1 {avail+=$4} END {print avail}'
+}
```

Comment:
echo was useless here. Suggested by shellcheck.
Same below in `formatByteCount`